yt/frontends/tipsy/data_structures.py

```diff
@@ -20,9 +20,7 @@
 from .fields import \
     TipsyFieldInfo
 
-import sys
-if sys.version_info > (3,):
-    long = int
+long = int
```

Comment:
We should get rid of any occurrence of `long` in the following of the file.

Reply:
This is done in #2640. I think these PRs should be merged together, but I forgot to mention it !

Reply:
It's ok, you don't need to merge them together! Noting they're complimentary in the comments here or the PR works too (and smaller ones are easier to review anyways).

Reply:
Indeed, my bad I went through this PR before #2640!